spec/factories/top_collection_metric.rb

```diff
@@ -0,0 +1,14 @@
+# The majority of the Supplejack API code is Crown copyright (C) 2014, New Zealand Government,
```

Comment:
remove this comment, we don't put the licence in anymore

Reply:
we sure don't 👍